src/components/core/DpDataTable/DpSelectPageItemCount.vue

```diff
@@ -29,14 +30,20 @@ export default {
       required: true
     },
 
+    labelText: {
+     type: String,
+     required: true
+    },
+
     pageCountOptions: {
       type: Array,
       required: true
     },
 
-    labelText: {
+    selectId: {
```

Comment:
Why do we need to set this dynamically? I'd make it a computed instead.

Reply:
like so? e16ace514de60e25f9ed320cbc145926a783f5da